src/storage/txn/process.rs

```diff
@@ -106,18 +108,20 @@ pub trait MsgScheduler: Clone + Send + 'static {
 
 pub struct Executor<E: Engine, S: MsgScheduler> {
     // We put time consuming tasks to the thread pool.
-    sched_pool: Option<SchedPool<E>>,
+    sched_pool: Option<SchedPool>,
     // And the tasks completes we post a completion to the `Scheduler`.
     scheduler: Option<S>,
     // If the task releases some locks, we wake up waiters waiting for them.
     waiter_mgr_scheduler: Option<WaiterMgrScheduler>,
     detector_scheduler: Option<DetectorScheduler>,
+
+    _phantom: PhantomData<E>,
```

Comment:
Seems this member never used?

Reply:
Yes, and it won't hold any space, just make it easier to compile. Without it we need to remove all `<E>` from `Executor`.